deploy/config/server.conf

```diff
@@ -34,7 +34,6 @@ server {
 
     client_max_body_size 250m;
     error_page 400 /app/themes/justice/error-pages/400.html;
```

Comment:
It looks like a mismatch between `400.html` here and the file `401.html`.

Same in `deploy/config/local/server.conf`